### PR TITLE
feat: [Helm] Generate password for Keycloak

### PR DESCRIPTION
**Pull request by mmorhun**
Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

### What does this PR do?
Generates random password for Keycloak admin user and prints it into console on Che installatin step.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/14082

### Depends on
https://github.com/eclipse/che/pull/16849

### CODE REVIEW

File: src/tasks/installers/helm.ts [hunk 1]
```diff
@@ -333,6 +334,13 @@ error: E_COMMAND_FAILED`)
       setOptions.push(`--set global.chePostgresPVCStorageClassName=${flags['postgres-pvc-storage-class-name']}`)
     }
 
+    if (flags.multiuser) {
+      // Generate Keycloak admin password
+      const keycloakPassword = generatePassword(12)
+      setOptions.push(`--set che-keycloak.keycloakAdminUserPassword=${keycloakPassword}`)
+      ctx.highlightedMessages.push(`Autogenerated Keycloak credentials is: "admin:${keycloakPassword}"`)
```
Comment (tolusha) [inline src/tasks/installers/helm.ts:341]: is -> are

File: src/tasks/installers/common-tasks.ts [hunk 2]
```diff
@@ -88,7 +88,7 @@ export function createEclipeCheCluster(flags: any, kube: KubeHelper): ListrTask
         }
 
         if (cr.spec.auth && cr.spec.auth.updateAdminPassword) {
-          ctx.highlightedMessages.push('You will be asked to change the default Che admin password on the first login.')
+          ctx.highlightedMessages.push('Che admin credentials are: "admin:admin". You will be asked to change default Che admin password on the first login.')
```
Comment (tolusha) [inline src/tasks/installers/common-tasks.ts:91]: Che -> Eclipse Che

File: src/tasks/installers/helm.ts (hunk 1 above)
Comment (tolusha) [reply src/tasks/installers/helm.ts:341]: `The autogenerated`

Comment (tolusha) [reply src/tasks/installers/helm.ts:341]: Could you provide the full `chectl` output?

Comment (mmorhun) [reply src/tasks/installers/helm.ts]:
```
  ✔ Show important messages
    ✔ ❗[MANUAL ACTION REQUIRED] Please add Che self-signed CA certificate into your browser: /home/mmorhun/cheCA.crt.
   Documentaton how to add a CA certificate into a browser: https://www.eclipse.org/che/docs/che-7/installing-che-in-tls-mode-with-self-signed-certificates/#using-che-with-tls_installing-che
-in-tls-mode-with-self-signed-certificates
    ✔ Autogenerated Keycloak credentials are: "admin:42uE9uFZdGZZ"
Command server:start has completed successfully.
```

**Comment by mmorhun** [comment]
Rebased

**Pull request merged**